src/openzaak/conf/includes/api.py

```diff
@@ -1,12 +1,20 @@
 # SPDX-License-Identifier: EUPL-1.2
 # Copyright (C) 2019 - 2022 Dimpact
-from vng_api_common.conf.api import *  # noqa - imports white-listed
+from vng_api_common.conf.api import (  # noqa: F401
+    BASE_REST_FRAMEWORK,
+    BASE_SWAGGER_SETTINGS,
+    COMMON_SPEC,
+    ZDS_CLIENT_CLASS,
+    GEMMA_URL_TEMPLATE,
+    GEMMA_URL_COMPONENTTYPE,
+    GEMMA_URL_INFORMATIEMODEL,
```

Comment:
Correct me if I'm wrong but these settings are used only in generate_swagger command which is outdated and not used in OZ (and will be removed together with drf-yasg)